pkg/options/validate.go

```diff
@@ -41,8 +44,19 @@ func (r ReleaseOptions) Append(chartName string, chartVersion string) ReleaseOpt
 	if !ok {
 		versions = []string{}
 	}
+
+	if slices.Index(versions, chartVersion) != -1 {
+		// value is present, do not include it
+		return r
+	}
+
 	versions = append(versions, chartVersion)
+
+	// keep sorted by semver
+	slices.SortFunc(versions, CompareVersions)
+
```

Comment:
Should we keep it sorted on every append, or should we just call `releaseYaml.SortBySemver()` when we actually dump the release.yaml?

I like the latter since it might be more efficient, unless there's a reason you'd prefer to keep the ReleaseOptions always sorted!

Reply:
Yeah that's fair. I have no good reason for keeping sorted, other than expecting the output to be sorted.

Reply:
I removed this and instead added a new call to `SortBySemver()` before dumping the contents into `release.yaml`.